pkgs/development/python-modules/flake8/default.nix

```diff
@@ -51,6 +52,8 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabled = !isPy3k;
```

Comment:
Please use pythonOlder because only 3.6 and higher is supported.

Reply:
Done, thx.